sycl/doc/GetStartedGuide.md

```diff
@@ -566,8 +566,8 @@ Creating a file `simple-sycl-app.cpp` with the following C++/SYCL code:
 #include <sycl/sycl.hpp>
 
 int main() {
-  // Creating buffer of 4 ints to be used inside the kernel code
-  sycl::buffer<sycl::cl_int, 1> Buffer(4);
+  // Creating buffer of 4 elements to be used inside the kernel code
+  sycl::buffer<size_t, 1> Buffer(4);
```

Comment:
Just out of curiosity: why did you choose `size_t` (`int` or `int32_t` seems to be more obvious replacement for `cl_int`)?

Reply:
That's what `sycl::id<1>::get(size_t)` returns. We assign it to this accessor at line 586.